napari_tiledb_bioimg/_reader.py

```diff
@@ -24,33 +24,46 @@ def napari_get_reader(path):
         return None
 
     config = None
+    valid = False
+    if os.path.exists(path):
+        valid=True
+    else: 
+        warnings.warn(f"Not a local valid path {path}")
+    
     if path.startswith('tiledb://'):
         # This tiledb paths should be resolved with cloud creds found under $HOME./tiledb/cloud.json
         # the current spec of napari_get_reader does not allow parameter passing
         # Read through tiledb-cloud
+        valid=True
         try:
           import tiledb.cloud
           config = tiledb.cloud.Config()
         except ImportError as exc:
             raise ImportError("[TileDB:Napari:BioImg] TileDB URIs require installation of tiledb-cloud") from exc
+    
+    if path.startswith('s3://'):
+        valid=True
 
-    # Scope with ctx
-    with tiledb.scope_ctx(ctx_or_config=config):
-        if tiledb.object_type(path) != "group":
-            warnings.warn(f"Not a tiledb group: {path}")
-            return None
+    if valid:
```

Comment:
Should we error out earlier rather than putting this whole section in an `if` block?